tools/mod/go.mod

```diff
@@ -161,7 +161,7 @@ require (
 	github.com/quasilyte/regex/syntax v0.0.0-20210819130434-b3f0c404a727 // indirect
 	github.com/quasilyte/stdinfo v0.0.0-20220114132959-f7386bf02567 // indirect
 	github.com/rivo/uniseg v0.4.7 // indirect
-	github.com/ryancurrah/gomodguard v1.3.2 // indirect
+	github.com/ryancurrah/gomodguard v1.3.3 // indirect
```

Comment:
This is purely indirect. Normally we would not bump it manually unless the direct dependency updates it.

Can you please clarify why we are bumping it?

Reply:
Yes I missed it, I will revert the indirect bumps. Thank you!